SwrveSDKDemo/build.gradle

```diff
@@ -1,12 +1,12 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 22
-    buildToolsVersion '22.0.1'
+    compileSdkVersion 23
+    buildToolsVersion '21.1.2'
```

Comment:
Yeah Dom had said he was having issues. I'd like to use 23.01 across the board. 

I'll make a point to upgrade them today and jira up any issues I come across